sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -145,6 +147,16 @@ export class ApplicationService extends RecordDataModelService<Application> {
     originalAssessment.submittedBy = auditUser;
     originalAssessment.submittedDate = now;
     originalAssessment.creator = auditUser;
+    const institutionLocation =
+      await this.institutionLocationService.getInstitutionLocation(
+        applicationData.selectedLocation,
+      );
+    if (!institutionLocation) {
+      throw new CustomNamedError(
+        "Not able to find the institution location.",
+        INSTITUTION_LOCATION_NOT_VALID,
```

Comment:
This error must be included in the controller to be transformed appropriately. 

![image](https://user-images.githubusercontent.com/54600590/232648914-549d28b4-618e-4ea7-b115-7237e5610bc4.png)